src/scry/context.cr

```diff
@@ -143,6 +143,7 @@ module Scry
       case msg.method
       when "textDocument/didClose"
         @workspace.drop_file(params)
+        return PublishDiagnostic.new(@workspace, params.text_document.uri).full_clean
```

Comment:
I'd better use:
```cr
case msg.method
when "textDocument/didClose"
  @workspace.drop_file(params)
  PublishDiagnostic.new(@workspace, params.text_document.uri).full_clean
else
  nil
end
```
to simplify control flow

Reply:
Done! :+1: